xsd/optaplanner/index.html.haml

```diff
@@ -6,7 +6,7 @@ priority: 0.1
 ---
 %h1 #{page.title}
 %ul
-  - for file in Dir.entries("xsd").sort
+  - for file in Dir.entries("xsd/optaplanner").sort
```

Comment:
Specifically moved the XSD to `xsd/optaplanner` to match its namespace:
https://www.optaplanner.org/xsd/optaplanner

Reply:
I presume that the links on the generated page in the _site output still work?

Reply:
Are there any links pointing to this page? I haven't found any and accessed directly to the built website via URL

Reply:
Perfect :)